README.md

```diff
@@ -81,142 +81,11 @@ Note that kpod and its container management and debugging commands have moved to
 
 For async communication and long running discussions please use issues and pull requests on the github repo. This will be the best place to discuss design and implementation.
 
-For sync communication we have an IRC channel #CRI-O, on chat.freenode.net, that everyone is welcome to join and chat about development.
+For sync communication we have an IRC channel #CRI-O on chat.freenode.net, and a [channel on the kubernetes slack](https://kubernetes.slack.com/archives/crio) that everyone is welcome to join and chat about development.
```

Comment:
I know it wasn't you, but I'd suggest "chat communication" over "sync communication"